src/main/java/dk/dbc/commons/slowcalllog/NanoUnit.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Det virker som lidt overkill at have alle disse units til dette formål. Hvorfor ikke bare angive tallet i f.eks. fractionelle ms? I praksis dækker det vel alt hvad man vll have brug for?